build/installers/liferay-developer-studio/liferay-developer-studio.xml

```diff
@@ -397,6 +397,35 @@
                 </compareValues>
             </ruleList>
         </propertiesFileSet>
+        <createDirectory>
+            <path>${lrws}${platform_path_separator}configs${platform_path_separator}common${platform_path_separator}deploy</path>
+            <ruleList>
+                <compareValues>
+                    <logic>equals</logic>
+                    <value1>${isdxp}</value1>
+                    <value2>dxp</value2>
+                </compareValues>
+                <fileTest>
+                    <condition>exists</condition>
+                    <path>${activationKey}</path>
+                </fileTest>
+            </ruleList>
+        </createDirectory>
+        <copyFile>
+             <destination>${lrws}${platform_path_separator}configs${platform_path_separator}common${platform_path_separator}deploy${platform_path_separator}activation_key.xml</destination>
```

Comment:
The name of the destination file should be the same name as the file that the user added.  We don't need to rename it to `activation_key.xml`